frontend/src/libraries/explorer-wamp/contracts.ts

```diff
@@ -26,12 +26,19 @@ export default class ContractsApi extends ExplorerApi {
           ]).then((info) => info[0]),
           this.queryAccessKey(id),
         ]);
-        return {
-          codeHash: codeHash,
-          transactionHash: contractInfo.hash,
-          timestamp: contractInfo.block_timestamp,
-          accessKeys: accessKeys.keys,
-        };
+        if (contractInfo !== undefined) {
+          return {
+            codeHash,
+            transactionHash: contractInfo.hash,
+            timestamp: contractInfo.block_timestamp,
+            accessKeys: accessKeys.keys,
+          };
+        } else {
+          return {
```

Comment:
Let's mark the function to actually return the `Promise<ContractInfo>` and fix the interface definition as definitely don't match this return value.